docs/src/index.md

```diff
@@ -307,23 +307,25 @@ using CSV
 CSV.write("c:/data/a.csv", a)
 
 # Windows users who do not have gzip available on the PATH should manually gzip the CSV
-;gzip a.csv
+;gzip c:/data/a.csv
 
-using CodecZlib
-@time a= CSV.read(GzipDecompressorStream(open("c:/data/a.csv.gz")))
+using CodecZlib, BufferedStreams
+io = open("c:/data/a.csv.gz") |> GzipDecompressorStream |> BufferedInputStream
```

Comment:
I'd prefer something more idiomatic Julia, like:
```julia
@time a = open("c:/data/a.csv.gz") do io
    CSV.read(GzipDecompressorStream(io))
end
```

Reply:
I F up by deleting the branch T_T. Do you think you can finish off the PR?

Reply:
Also I don't understand why you introduce Windows-specific paths here.

Reply:
Just an example. Was easier to test as I was writing it

Reply:
Yes but we'd better keep things OS-neutral when possible.